<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordSet</class>
 <widget class="QWidget" name="PasswordSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 18px, &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:1px solid white;
	border-radius: 5px;
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>125</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>原密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>195</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>265</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1Origin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>125</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(0, 0, 255);
border:2px solid white;
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1New">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>195</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: black;
border-radius: 5px;
border:2px solid white;


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1Confirm">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>265</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: black;
border:2px solid white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1Tips">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>335</y>
     <width>291</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
border: 0px;</string>
   </property>
   <property name="text">
    <string>提示信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1Set">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>335</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Confirm">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>265</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: black;
border:2px solid white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>265</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2Tips">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>335</y>
     <width>291</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
border: 0px;</string>
   </property>
   <property name="text">
    <string>提示信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Set">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>335</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>设定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>125</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>原密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>195</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Origin">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>125</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: black;
border:2px solid white;
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2New">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>195</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: black;
border:2px solid white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_106">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>75</y>
     <width>451</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_107">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>75</y>
     <width>451</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: white;
	border: 0px groove, gray;
    border-color: gray;
	font: 20px, &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>检修登录密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: white;
	border: 0px groove, gray;
    border-color: gray;
	font: 20px, &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>数据设置密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_107</zorder>
  <zorder>label_106</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>btn1Origin</zorder>
  <zorder>btn1New</zorder>
  <zorder>btn1Confirm</zorder>
  <zorder>lbl1Tips</zorder>
  <zorder>btn1Set</zorder>
  <zorder>btn2Confirm</zorder>
  <zorder>label_6</zorder>
  <zorder>lbl2Tips</zorder>
  <zorder>btn2Set</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>btn2Origin</zorder>
  <zorder>btn2New</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
